recipes/npcap/all/test_package/conanfile.py

```diff
@@ -31,17 +30,13 @@ def build(self):
 
     def test(self):
         if can_run(self):
-            bindir = self.cpp.build.bindirs[0]
+            bindir = self.cpp.build.bindir
             # Use libpcap DLL as a replacement for npcap DLL
             # It will not provide all the functions
             # but it will cover enough to check that what we compiled is correct
             files.rm(self, "wpcap.dll", bindir)
-            files.copy(self, "pcap.dll", src=self.deps_cpp_info['libpcap'].bin_paths[0], dst=bindir)
+            libpcap_bin_path = self.deps_cpp_info["libpcap"].bin_paths[0] if conan_version < "2" else self.dependencies["libpcap"].cpp_info.bindirs[0]
```

Comment:
This check should not be necessary. V1 also supports `self.dependencies` attribute

Reply:
That's what I thought too, but Conan v1 pipeline failed saying that `self.dependencies` was not available.
Unfortunately I don't have the logs because the current CI report overrides the previous one. Maybe you have access to it?

Reply:
Here is the commit where I changed it: https://github.com/conan-io/conan-center-index/pull/20399/commits/320dfaab399c6f9819a86acbcfcaa0a89cff7710

Reply:
@seladb Please, avoid checking conan version and use only `self.dependencies`. As workaround for Conan 1.x you can use: https://github.com/conan-io/conan-center-index/blob/master/recipes/opentelemetry-proto/all/test_package/conanfile.py

Store the value in a file, then get it back in `test()`

Reply:
@uilianries @RubenRBS thanks for your review! Addressed here: https://github.com/conan-io/conan-center-index/pull/20399/commits/ea3ff475dd78e1dc52e98231f3353131a552cad5

Reply:
Oops, sorry about the confusion, I should have made myself more clear along the lines of what @uilianries did, thanks a lot for addressing the review so quickly :)